website/docs/03_features/03_configuration.mdx

```diff
@@ -0,0 +1,401 @@
+import TerminalWindow from '@site/src/components/TerminalWindow/TerminalWindow'
+
+# Configuration Management
+
+Booster provides a flexible configuration management system that allows you to retrieve configuration values from multiple sources with automatic fallback mechanisms. This enables you to manage application settings dynamically without code changes or redeployments.
+
+## Configuration Resolution Hierarchy
+
+Booster resolves configuration values using a 3-tier priority system (highest to lowest priority):
+
+1. **External Configuration Providers** - Azure App Configuration, custom providers (highest priority)
+2. **Booster config.env** - Configuration defined in your Booster config
+3. **System Environment Variables** - Standard process.env values (lowest priority)
+
+When you request a configuration value, Booster checks each source in order and returns the first value found.
+
+```typescript
+import { Booster } from '@boostercloud/framework-core'
+import { resolveConfigurationWithSource } from '@boostercloud/framework-core'
+
+// Resolve a configuration value with source tracking
+const resolution = await resolveConfigurationWithSource(Booster.config, 'API_TIMEOUT')
+console.log(`Value: ${resolution.value}, Source: ${resolution.source}`)
+```
```

Comment:
I would put another example with the other function in case the source of the variable isn't important for the programmer